101-machine-learning-compute-create-amlcompute/azuredeploy.json

```diff
@@ -0,0 +1,95 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {        
+        "workspaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Specifies the name of the Azure Machine Learning Workspace which will contain this compute."
+            }
+        },
+        "clusterName": {
+            "type": "string",
+            "metadata": {
+                "description": "Specifies the name of the Azure Machine Learning Compute cluster."
+            }
+        },        
+        "minNodeCount": {
+            "defaultValue": 0,
+            "type": "int",
+            "metadata": {
+                "description": "The minimum number of nodes to use on the cluster. If not specified, defaults to 0"
+            }
+        },        
+        "maxNodeCount": {
+            "defaultValue": 1,
+            "type": "int",
+            "metadata": {
+                "description": " The maximum number of nodes to use on the cluster. If not specified, defaults to 4."
+            }
+        },
+        "workspacelocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The location of the Azure Machine Learning Workspace."
+            }
+        },
+        "adminUserName": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The name of the administrator user account which can be used to SSH into nodes. It should only contain small case characters."
+            }
+        },
+        "adminUserPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The password of the administrator user account."
+            }
+        },
+        "vmSize": {
+            "defaultValue": "Standard_DS3_v2",
+            "type": "string",
+            "metadata": {
+                "description": " The size of agent VMs. More details can be found here: https://aka.ms/azureml-vm-details."
+            }
+        },
+        "subnetId": {
```

Comment:
break this apart into multiple parameters, one for the vnet resourceGroup, vnet name, and subnet name...